src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
+
+const convertToNumber = (str) => Number(str.replace('$', '').replace(',', ''));
```

Comment:
The function `convertToNumber` is only converting strings that have a '$' and ',' characters. It may not work correctly if the string has other non-numeric characters. Consider using a more robust conversion function that removes all non-numeric characters.